### PR TITLE
tests: benchmarks: power_consumption: adc: multicore app for nrf54h20

### PR DESCRIPTION
**Pull request by nordic-piks**
Add code for radio core so that it can be put into sleep and release resources.

### CODE REVIEW

**Review by nordic-bami — APPROVED** [review]

**Review by nordic-segl — APPROVED** [review]

**Review by nordic-babu — APPROVED** [review]

**Review by nordic-pikr — APPROVED** [review]

File: tests/benchmarks/power_consumption/adc/sysbuild.cmake [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(NOT "${SB_CONFIG_REMOTE_BOARD}" STREQUAL "")
+  ExternalZephyrProject_Add(
+    APPLICATION remote_sleep_forever
+    SOURCE_DIR ${APP_DIR}/../common/remote_sleep_forever
+    BOARD ${SB_CONFIG_REMOTE_BOARD}
+  )
```
Comment (nordicjm) [inline tests/benchmarks/power_consumption/adc/sysbuild.cmake:12]: missing BOARD_REVISION

File: tests/benchmarks/power_consumption/adc/sysbuild.cmake [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  add_dependencies(adc remote_sleep_forever)
+  sysbuild_add_dependencies(FLASH adc remote_sleep_forever)
+
```
Comment (nordicjm) [inline tests/benchmarks/power_consumption/adc/sysbuild.cmake:16]:
```suggestion
```

**Review by nordicjm — CHANGES_REQUESTED** [review]
Apply changes to each commit

File: tests/benchmarks/power_consumption/adc/sysbuild.cmake (hunk 1 above)
Comment (nordic-piks) [reply tests/benchmarks/power_consumption/adc/sysbuild.cmake:13]: Fixed.

File: tests/benchmarks/power_consumption/adc/sysbuild.cmake (hunk 2 above)
Comment (nordic-piks) [reply tests/benchmarks/power_consumption/adc/sysbuild.cmake]: Fixed.

**Review by nordicjm — APPROVED** [review]

**Pull request merged**